packages/docs-site/src/library/pages/components/phase-banner.md

```diff
@@ -51,7 +51,7 @@ By default simply passing a `label` and `href` will cause a regular anchor tag t
 </CodeHighlighter>
 
 ### Alternative text
-You can pass custom markup to appear by including it as a child of the component.
+You can pass custom mark-up to appear by including it as a child of the component.
```

Comment:
I feel in the context of technology [it is "markup"](https://en.wikipedia.org/wiki/HTML).